### DISCUSSION TITLE
Default width when outputting to PDF

### DISCUSSION THREAD

**Discussion by snhansen** [rstudio/gt#1857] [Q&A]
With the new way of creating tables for PDF ouput (using table/tabular instead of longtable), I find it a bit weird that if I specify `cols_width()` using `px()`, I still get a full-width table, and not a table with a width equal to the sum of `px()`-statements.

Example:

```
mtcars[1:4, 1:2] |>
  gt() |>
  cols_width(
    mpg ~ px(100),
    cyl ~ px(150)
  ) |>
  as_latex()
```

and 

```
mtcars[1:4, 1:2] |>
  gt() |>
  cols_width(
    mpg ~ px(100),
    cyl ~ px(150)
  ) |>
  tab_options(
    table.width = 250
  ) |>
  as_latex()
```

yield two different tables. Would it be an idea to set the `table.width`-argument to the sum of widths-statements if not otherwise specified?

**Comment by olivroy** [comment]
@AaronGullickson @nielsbock @kbrevoort You may want to weigh in here! Now is still a good time to change this since the tabular output was just introduced. #1852 also of interest

**Reply by kbrevoort** [reply]
@olivroy, the border control PR that I mentioned yesterday in #1864 includes a fix for this. I'll try to finish that PR soon, but as I mentioned, I may not be able to get back to it before early October.
